pages/activityList.js

```diff
@@ -35,26 +36,127 @@ const StyledLi = styled.li`
   padding: 20px 0 20px 0;
 `;
 
+const StyledSection = styled.section`
+  display: flex;
+  flex-direction: column;
+  gap: 0.6rem;
+`;
```

Comment:
To have a better overview here use a seperate file for styled components.

Reply:
We think that is a really good suggestion and we definitely want to re-structure our styles and logic better in the future by outsourcing the styled components 😊 But we would like to do this as a separate User Story as we want to do this outsourcing in one step for every page/component 👍